src/activities/calendar/Calendar.qml

```diff
@@ -0,0 +1,459 @@
+
+        Rectangle {
+            id: questionItemBackground
+            color: "black"
+            border.width: 2
+            radius: 10
+            opacity: 0.85
+            z: 10
+            anchors {
+                horizontalCenter: parent.horizontalCenter
+                bottomMargin: 10
+            }
+            width: items.horizontalLayout ? calendarBox.width * 2 : parent.width - 20
+            height: calendarBox.height * 0.125
+            gradient: Gradient {
+                GradientStop { position: 0.0; color: "#000" }
+                GradientStop { position: 0.9; color: "#666" }
+                GradientStop { position: 1.0; color: "#AAA" }
+            }
+        }
+
+        // Displays the question.
+        GCText {
+            id: questionItem
+            anchors.fill: questionItemBackground
+            anchors.bottom: questionItemBackground.bottom
+            fontSizeMode: Text.Fit
+            wrapMode: Text.Wrap
+            z: 10
+            color: "white"
+            verticalAlignment: Text.AlignVCenter
+            horizontalAlignment: Text.AlignHCenter
+        }
+
+        // Answer Submission button.
+        BarButton {
+            id: okButton
+            source: "qrc:/gcompris/src/core/resource/bar_ok.svg"
+            width: 50 * ApplicationInfo.ratio
+            height: width
+            sourceSize.width: width
+            sourceSize.height: height
+            y: parent.height * 0.8
+            z: 10
+            anchors {
+                horizontalCenter: score.horizontalCenter
+                bottom: calendarBox.bottom
+            }
+            ParticleSystemStarLoader {
+                id: okButtonParticles
+                clip: false
+            }
+            MouseArea {
+                id: okButtonMouseArea
+                anchors.fill: parent
+                onClicked: {
+                    Activity.checkAnswer()
+                }
+            }
+        }
+
+        DialogHelp {
+            id: dialogHelp
+            onClose: home()
+        }
+
+        Bar {
+            id: bar
+            content: BarEnumContent { value: help | home | level }
+            onHelpClicked: {
+                displayDialog(dialogHelp)
+            }
+            onPreviousLevelClicked: Activity.previousLevel()
+            onNextLevelClicked: Activity.nextLevel()
+            onHomeClicked: activity.home()
+        }
+
+        Bonus {
+            id: bonus
+            Component.onCompleted: win.connect(Activity.nextLevel)
+        }
+
+        Score {
+            id: score
+            fontSize: items.horizontalLayout ? 16 : (answerChoices.visible ? 12 : 8)
+            anchors.top: calendarBox.top
+            anchors.bottom: undefined
+            anchors.left: calendarBox.right
+            anchors.right: undefined
+            anchors.margins: items.horizontalLayout ? 30 : 10
+            property alias winAnimation: winAnmations
+            SequentialAnimation {
```

Comment:
if it can be used on other activities, it would be better to have it directly in Score.qml and have a signal to trigger it